vm/chef/cookbooks/php73/attributes/default.rb

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+default['php73']['packages'] = ['php7.3-fpm', 'php7.3-common', 'php7.3']
```

Comment:
Why the `php7.3-common` package is installed by default?

In the PHP 7.0 it was installed as "default modules". Is there any reason why this was changed?

Reply:
I installed it by default because it's a dependency to php 7.3 according to https://packages.debian.org/buster/php7.3